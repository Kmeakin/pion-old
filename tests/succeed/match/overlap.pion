//~ run = "${PION} elab ${FILE}"

fun (x : Bool) => match x {
    false => 1,
    true => 0,
    false => 1,
    true => 0,
}
