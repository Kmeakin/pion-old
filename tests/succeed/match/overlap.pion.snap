exit status: 0
stdout:
fun (x : Bool) => match x {
    false => 1,
    true => 0,
} : Bool -> Int

stderr:
warning: unreachable pattern
  ┌─ tests/succeed/match/overlap.pion:6:5
  │
6 │     false => 1,
  │     ^^^^^

warning: unreachable pattern
  ┌─ tests/succeed/match/overlap.pion:7:5
  │
7 │     true => 0,
  │     ^^^^

